admin/includes/languages/english/lang.banner_manager.php

```diff
@@ -14,6 +14,7 @@
     'TABLE_HEADING_BANNER_OPEN_NEW_WINDOWS' => 'New Window',
     'TABLE_HEADING_BANNER_ON_SSL' => 'Show SSL',
     'TABLE_HEADING_BANNER_SORT_ORDER' => 'Sort<br>Order',
+    'TEXT_BANNERS_INTRO' => 'Each Banner (text or image) is assigned to a Banner Group along with other Banners, to allow different Banners to be displayed (rotated) in the same position.<br>The position of a Banner Group on the shopfront page is defined in <a href="index.php?cmd=' . FILENAME_CONFIGURATION . '&amp;gID=19&amp;cID=427"> Configuration->Layout Settings</a>.<br>You may create a new Banner Group when creating/editing a Banner.<br>Use the Help icon <strong>?</strong> for more details.',
```

Comment:
Might be better not to change a language constant in this way since it will break overrides.  Just change the caller to use `zen_href_link` as Cindy suggested rather than trying to build the URL.

Reply:
done